.gitignore

```diff
@@ -1,5 +1,9 @@
 # OSX
 .DS_Store
+.idea/
+
+# VS Code
+.vscode
```

Comment:
In other Repos we have `# IDEs`. `.idea/` is not OSX specific, but IntelliJ IDE specific 🤔
Can be updated later.